### PR TITLE
Add tangent

### PR DESCRIPTION
**Pull request by Codesuckmehard**
To calculate the radians of tangent

### CODE REVIEW

File: math/tan.go [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package math
+
+import (
+	"math"
+)
+
+func Tan(x float64) float64 {
+	return Cot((math.Pi / 2) - x)
```
Comment (tjgurwara99) [inline math/tan.go:8]: I don't think we have `Cot` defined, do we?

Anyways, we have `Sin` and `Cos` defined so why not use the `Sin/Cos`?

**Review by tjgurwara99 — CHANGES_REQUESTED** [review]

**Comment by raklaptudirm** [comment]
IMO, this algorithm is too trivial.

**Comment by tjgurwara99** [comment]
> IMO, this algorithm is too trivial.

Depends on what you mean by trivial. If you look at our [Cos](https://github.com/TheAlgorithms/Go/blob/feb2246c4605f9c9b4e00a491b2930524bbda2be/math/cos.go#L10) then I don't think composing these algorithms together to make new ones is trivial. If this is trivial, then our [Sin](https://github.com/TheAlgorithms/Go/blob/feb2246c4605f9c9b4e00a491b2930524bbda2be/math/sin.go#L9) is also quite trivial.

**Review by raklaptudirm — CHANGES_REQUESTED** [review]
Format your code with <samp>gofmt</samp>.

**Pull request closed without merging**